plutus-tutorial/tutorial/Tutorial/03-wallet-api.md

```diff
@@ -1,17 +1,20 @@
-# wallet-api: Wallet-API tutorial
+# Wallet API II: Smart Contracts
 
-This tutorial shows how to implement a simple crowdfunding campaign as a Plutus contract, using the wallet API to submit it to the blockchain. There are two ways to run the code:
+This tutorial shows how to implement a crowdfunding campaign as a Plutus contract, using the wallet API to submit it to the blockchain. It is the third of three tutorials on Plutus smart contracts. The [first tutorial](./01-plutus-tx.md) explains how to use Template Haskell and the Plutus compiler to write programs that are _blockchain-ready_. The [second tutorial](./02-validator-scripts.md) shows how to get those programs onto the blockchain.
 
-1. Open the [Plutus Playground](https://prod.playground.plutus.iohkdev.io/), delete all the text in the editor field, and type / copy the code bits in there. Make sure to preserve the indentation.
-2. The tutorial is written as a literate Haskell file, so it can be fed directly to the Haskell compiler. The easiest way to do that is to clone the Plutus repository at `git@github.com:input-output-hk/plutus.git` and run `nix build -f default.nix localPackages.plutus-tutorial`. This command builds the `plutus-tutorial` nix derivation, which includes this tutorial. Alternatively, run `cabal test plutus-tutorial`. Note that the test suite requires Unix symlinks to be supported by the file system, which means that it will not work on Windows Subsystem for Linux (WSL), even though nix generally does work!
+You can run this code in the [Plutus Playground](https://prod.playground.plutus.iohkdev.io/) - see Section 2.1, "Testing the contract in the Playground". 
 
-We assume the reader is familiar with the [UTxO model with scripts](../../../docs/extended-utxo/README.md) and the [PlutusTx tutorial](./01-plutus-tx.md). Please note that the wallet API and by extension the wallet API tutorial is a work in progress and may be changed significantly without much warning. 
+There is another option to run the code: The tutorial is written as a literate Haskell file, so it can be fed directly to the Haskell compiler. The easiest way to do that is to clone the Plutus repository at `git@github.com:input-output-hk/plutus.git` and run `nix build -f default.nix localPackages.plutus-tutorial`. This command builds the `plutus-tutorial` nix derivation, which includes this tutorial. Alternatively, run `cabal test plutus-tutorial`. Note that the test suite requires Unix symlinks to be supported by the file system, which means that it will not work on Windows Subsystem for Linux (WSL), even though nix generally does work!
```

Comment:
They would need install nix first.

Reply:
I took this bit out and added a section on setup to the front page.